scalding-serialization/src/main/scala/com/twitter/scalding/serialization/macros/impl/ordered_serialization/providers/ImplicitOrderedBuf.scala

```diff
@@ -27,13 +27,11 @@ import com.twitter.scalding.serialization.macros.impl.ordered_serialization._
   type. This is for the case where its an opaque class to our macros where we can't figure out the fields
 */
 object ImplicitOrderedBuf {
-  val macroMarker = "MACROASKEDORDEREDSER"
```

Comment:
So this was being used just to form type aliases before? Was this some macro workaround that's no longer needed?